tools/cli/domainCommands.go

```diff
@@ -69,6 +75,10 @@ func newDomainCLI(
 			return c.String(FlagDestinationAddress)
 		}).ServerFrontendClient(c)
 	} else {
+		d.frontendAdminClient = initializeFrontendAdminClient(c)
```

Comment:
This requires admin mode set to true. Could you move the whole command to `newAdminDomainCommands`

Reply:
discussed offline. we will keep it here but enable admin mode.